frame/staking/src/pallet/impls.rs

```diff
@@ -668,9 +668,9 @@ impl<T: Config> Pallet<T> {
 
 	/// Clear all era information for given era.
 	pub(crate) fn clear_era_information(era_index: EraIndex) {
-		let _ = <ErasStakers<T>>::clear_prefix(era_index, u32::MAX, None);
-		let _ = <ErasStakersClipped<T>>::clear_prefix(era_index, u32::MAX, None);
-		let _ = <ErasValidatorPrefs<T>>::clear_prefix(era_index, u32::MAX, None);
+		debug_assert!(<ErasStakers<T>>::clear_prefix(era_index, u32::MAX, None).maybe_cursor.is_none());
```

Comment:
This will actually only run the statement in debug builds… so you have to put the cursor into a variable.